### PR TITLE
Explainer update to use .setHTML method.

### PR DESCRIPTION
**Pull request by otherdaniel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,10 +42,12 @@ If you wish to try out early Sanitizer implementations, the
...
-The core API of the Sanitizer is rather simple: There are `Sanitizer` objects,
-and they have a `.sanitize` method that produces a `DocumentFragment` from a
-given text string. Or from another `DocumentFragment`, or even from a whole
-`Document`.
+The core API of the Sanitizer is rather simple: There are `Sanitizer` objects
+which encapsulate the information of what the sanitizer is supposed to do,
+and there is an `Element.setHTML` method which takes a string and (optionally)
+an options dictionary with a `"sanitizer"` key and a `Sanitizer` instance as
+a value. `.setHTML` will parse and sanitize the string it receives, and will
+update the children of the Element it is called on with the result.
```
Comment (mozfreddyb) [inline README.md:50]: Quite the long sentences. Can we cut the complexity by making it grow gradually?
How about something along those lines:
* This API prevents XSS, which means sanitizing a tree of DOM nodes
* There's a safe default that states which should be removed and kept
* if all you care about is XSS, you may just use `someElement.setHTML(untrusted_input)`
* if you need to customize, use a Sanitizer constructor to hold the configuration and pass it in

Would that make sense?

File: README.md [hunk 2]
```diff
@@ -42,10 +42,12 @@ If you wish to try out early Sanitizer implementations, the
...
 
 Example:
 ```js
```
Comment (mozfreddyb) [inline README.md:53]: While we're at it, I think we could start explaining/defining two variables for an element and for untrusted input a priori. Repeating the `document.querySelector()` call so often is a bit noisy, isn't it?

File: README.md [hunk 3]
```diff
@@ -92,14 +99,19 @@ Example:
   });
 
   const untrusted_example = "Well, <em class=nonchalant onclick='alert(\'General Kenobi\');'><a href='https://obiwan.org/home.php'>hello there<a>!"
```
Comment (mozfreddyb) [inline README.md:101]: obiwan.org seems to be an existing site? Can we use obiwan.example?

File: README.md (hunk 2 above)
Comment (otherdaniel) [reply README.md:58]: Done.

File: README.md (hunk 3 above)
Comment (otherdaniel) [reply README.md]: Done.

(I can't find a page or whois entry to obiwan.org, but let's use the reserved .example to be safe.)

**Review by mozfreddyb — APPROVED** [review]
thank you!

**Pull request merged**